proto/cosmos/upgrade/v1beta1/query.proto

```diff
@@ -64,5 +64,7 @@ message QueryUpgradedConsensusStateRequest {
 // QueryUpgradedConsensusStateResponse is the response type for the Query/UpgradedConsensusState
 // RPC method.
 message QueryUpgradedConsensusStateResponse {
-  google.protobuf.Any upgraded_consensus_state = 1;
+  reserved 1;
+
+  bytes upgraded_consensus_state = 2;
```

Comment:
why this was changed?

Reply:
Sorry for not documenting it, but I believe the reason why is because then the upgrade module would have a dependency on the IBC module which we wanted to completely avoid. In order to return the correct `Any`, the upgrade module would need to understand what an IBC consensus state is, thus creating a dependency on the IBC module

Is there an issue with the new behaviour?

Reply:
It breaks the API. It seams that you were not correctly using this function - because Any was not used.

Reply:
Sure, this is just a convenience function for the cli. Relayers who perform the client upgrade functionality do ABCI queries to obtain the proofs necessary